README.md

```diff
@@ -2,6 +2,120 @@
 
 # prpl-server-node
 
-A Node implementation of the [PRPL](https://developers.google.com/web/fundamentals/performance/prpl-pattern/) pattern for serving Progressive Web Apps.
+An HTTP server for Node designed for serving [PRPL](https://developers.google.com/web/fundamentals/performance/prpl-pattern/) apps in production.
```

Comment:
Phrasing nit: think this will read better if you say "designed to serve" rather than "designed for serving".